README.md

```diff
@@ -14,7 +14,6 @@
 [![npm][npm]][npm-url]
 [![Build Status][build-status]][build-status-url]
 [![codecov][codecov-badge]][codecov-url]
-[![Dependencies][deps]][deps-url]
```

Comment:
Also need to remove these links `deps` and `deps-url` from the bottom.

Reply:
Could you fix that?